.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Create a .env file
         run: |
           cat << EOF > .env
-          MYSQL_HOST=mysql
+          MYSQL_HOST=localhost
```

Comment:
MYSQL HOST needs to be "mysql" as this .env is loaded into the docker compose.
Here mysql means the service name of MySQL

Reply:
Ah right